control-plane/pkg/reconciler/trigger/v2/triggerv2.go

```diff
@@ -155,32 +184,45 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, broker *eventing
 		},
 	}
 
+	// TODO: make keda annotation values configurable and maybe unexposed
+	expectedCg.Annotations = kedafunc.SetAutoscalingAnnotations(broker.Annotations)
```

Comment:
We should read annotations from the Trigger object since each trigger might want to have a different configs based on the subscriber service ability to process events.